src/app/pages/LiveRadioPage/index.test.jsx

```diff
@@ -6,34 +6,37 @@ import { matchSnapshotAsync } from '@bbc/psammead-test-helpers';
 import { ServiceContextProvider } from '#contexts/ServiceContext';
 import { RequestContextProvider } from '#contexts/RequestContext';
 import LiveRadioPage from '.';
-import amharicPageData from '#data/amharic/bbc_amharic_radio/liveradio';
+import afriquePageData from './fixtureData/afrique';
+import indonesianPageData from './fixtureData/indonesia';
+import gahuzaPageData from './fixtureData/gahuza';
 import * as analyticsUtils from '#lib/analyticsUtils';
 import { ToggleContextProvider } from '#contexts/ToggleContext';
-import getInitialData from '#app/routes/liveRadio/getInitialData';
 
-const Page = ({ pageData, isAmp = false }) => (
+const Page = ({ pageData, service = 'afrique', isAmp = false }) => (
   <BrowserRouter>
-    <ToggleContextProvider service="amharic" origin="https://www.test.bbc.com">
-      <ServiceContextProvider service="amharic">
+    <ToggleContextProvider service={service} origin="https://www.test.bbc.com">
+      <ServiceContextProvider service={service} lang="fr">
```

Comment:
Should the 'lang' here be dynamically populated from `pagedata.language` rather than hard coded?

Reply:
It shouldn't be hardcoded to 'fr' as we also have Gahuza and Indonesia test data in here but I don't believe it should be taken from pagedata.language